lm_eval/datasets/hendrycks_math/hendrycks_math.py

```diff
@@ -38,8 +38,8 @@
 
 _HOMEPAGE = "https://github.com/hendrycks/math"
 
-# TODO: Add the licence for the dataset here if you can find it
-_LICENSE = ""
+# License available at https://github.com/hendrycks/math/blob/main/LICENSE
+_LICENSE = "MIT License"
```

Comment:
Likewise, unclear what license the actual data files (can be) distributed under, distinct from the codebase.

Reply:
I agree. I couldn't find the info on the MAA website and have emailed them asking.